examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java

```diff
@@ -89,7 +89,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     server.blockUntilShutdown();
   }
 
-  private class GreeterImpl extends GreeterGrpc.GreeterImplBase {
+  static class GreeterImpl extends GreeterGrpc.GreeterImplBase {
```

Comment:
I'm still of the opinion that classes named "HelloWorld" need to be as simple and clear as possible.  Testing does not belong in these classes.  They belong in some other class, with a name that doesn't imply that it is the minimum viable implementation.